contract/__manifest__.py

```diff
@@ -8,15 +8,15 @@
 
 {
     'name': 'Contracts Management - Recurring',
-    'version': '11.0.3.0.0',
+    'version': '11.0.4.0.0',
     'category': 'Contract Management',
     'license': 'AGPL-3',
     'author': "OpenERP SA, "
               "Tecnativa, "
               "LasLabs, "
               "Odoo Community Association (OCA)",
     'website': 'https://github.com/oca/contract',
-    'depends': ['base', 'account', 'analytic'],
+    'depends': ['base', 'account_invoicing', 'analytic'],
```

Comment:
Why this change?

Reply:
I can revert it if you like. But account_invoicing is the module that makes visible the invoicing menu.

Reply:
Well, the problem is to change dependencies on the middle of the lifecycle. You can put a comment for changing it on v12

Reply:
Oh, right. But v12 already merged <code>account_invoice</code> to <code>account</code> 😂